.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What is purpose of using `eslint`?
It would be better to define `pre-commit` hook that will run eslint for avoiding non-codestyle code in repo.
Take a look [pre-commit](https://www.npmjs.com/package/pre-commit) lib, please.
If you would like to implement it, just do it 😄 
If not just skip this comment.

FYI: Using eslint is out of scope homework task and will no be estimate.

Reply:
I've added =)
Not sure that I've done everything right, but anyway looks like it works!